QuickOSM/core/process.py

```diff
@@ -165,6 +166,10 @@ def open_file(
                 QgsExpressionContextUtils.setLayerVariable(
                     new_layer, 'quickosm_query', final_query)
 
+            metadata = QgsLayerMetadata()
+            metadata.setRights([tr("© OpenStreetMap contributors")])
+            metadata.setLicenses([tr('https://openstreetmap.org/copyright')])
```

Comment:
```suggestion
            metadata.setLicenses(['https://openstreetmap.org/copyright'])
```

Minor, but one string less to translate